SamplesCS/Samples/FaceDetectionDNN.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using OpenCvSharp;
+using OpenCvSharp.Dnn;
+using SampleBase;
+
+namespace SamplesCS
+{
+    /// <summary>
+    /// To run this example first download the face model available here:https://github.com/spmallick/learnopencv/tree/master/FaceDetectionComparison/models
+    /// Add the files to the bin folder
+    /// </summary>
+    class FaceDetectionDNN : ISample
+    {
+        public void Run()
+        {
+            string finalOutput = "DetectedFaces.jpg";
+            string faceModel = "res10_300x300_ssd_iter_140000_fp16.caffemodel";
+			string configFile = "deploy.prototxt";
+			var image = "faces.jpg";
+            // Read sample image
+            var frame = Cv2.ImRead(image);
+            int frameHeight = frame.Rows;
+            int frameWidth = frame.Cols;
+            using (var faceNet = CvDnn.ReadNetFromCaffe(configFile, faceModel)) 
+            {
+				using (Mat blob = CvDnn.BlobFromImage(frame, 1.0, new OpenCvSharp.Size(300, 300), new Scalar(104, 117, 123), false, false)) 
```

Comment:
decrease indent
```c#
using (var faceNet = CvDnn.ReadNetFromCaffe(configFile, faceModel)) 
using (Mat blob = CvDnn.BlobFromImage(frame, 1.0, new OpenCvSharp.Size(300, 300), new Scalar(104, 117, 123), false, false)) 
{
```

Reply:
But this is from the Face Detection Sample. Which was already merged.